internal/cloudconfig/cloudinit/cloudinit_ubuntu.go

```diff
@@ -178,16 +174,16 @@ func (cfg *ubuntuCloudConfig) getCommandsForAddingPackages() ([]string, error) {
 	cmds = append(cmds, config.PackageManagerLoopFunction)
 	looper := "package_manager_loop "
 
-	if cfg.SystemUpdate() {
-		cmds = append(cmds, LogProgressCmd("Running apt-get update"))
-		cmds = append(cmds, looper+pkgCmder.UpdateCmd())
-	}
-	if cfg.SystemUpgrade() {
-		cmds = append(cmds, LogProgressCmd("Running apt-get upgrade"))
-		cmds = append(cmds, looper+"apt-mark hold cloud-init")
-		cmds = append(cmds, looper+pkgCmder.UpgradeCmd())
-		cmds = append(cmds, looper+"apt-mark unhold cloud-init")
-	}
+	// if cfg.SystemUpdate() {
+	// 	cmds = append(cmds, LogProgressCmd("Running apt-get update"))
+	// 	cmds = append(cmds, looper+pkgCmder.UpdateCmd())
+	// }
+	// if cfg.SystemUpgrade() {
+	// 	cmds = append(cmds, LogProgressCmd("Running apt-get upgrade"))
+	// 	cmds = append(cmds, looper+"apt-mark hold cloud-init")
+	// 	cmds = append(cmds, looper+pkgCmder.UpgradeCmd())
+	// 	cmds = append(cmds, looper+"apt-mark unhold cloud-init")
+	// }
```

Comment:
We could in theory gut all of the apt stuff from cloud-init as, we no longer require it. Thus the package would be a lot cleaner.

Reply:
This isn't possible because we side-load additional packages for providers.